Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -175,7 +175,7 @@ local questsToBlacklistBySoDPhase = {
         [90078] = true, -- Hiding Warlock Demonic Grace Elwynn Forest for now as there are too many icons
         [90079] = true, -- Hiding Warlock Demonic Grace Durotar for now as there are too many icons
         [90080] = true, -- Hiding Warlock Demonic Grace Tirisfal Galdes for now as there are too many icons
-
+        [210187] = true, -- Hiding Rogue Deadly Brew Pre-req for now as there are too many icons
```

Comment:
Also this is the itemId, not the questId.
```suggestion
        [78270] = true, -- Hiding Rogue Deadly Brew Pre-req for now as there are too many icons
```